cmd/ops/deprecation.go

```diff
@@ -24,67 +23,72 @@ The "queries" argument is a file glob matching one or more graphql query or muta
 
 func init() {
 	Program.AddCommand(deprecationsCmd)
-	deprecationsCmd.Flags().StringVar(&flags.schemaFile, schemaFileFlagName, "", "Server's schema as file or url (required)")
+	deprecationsCmd.Flags().StringArrayVar(&flags.schemaFiles, schemaFileFlagName, []string{}, "Server's schema as file or url. Can be repeated (required)")
 	deprecationsCmd.MarkFlagRequired(schemaFileFlagName) //nolint:errcheck // will err if flag doesn't exist
-
-	deprecationsCmd.Flags().StringArrayVar(&flags.ignore, ignoreFlagName, []string{}, "Files to ignore")
+	deprecationsCmd.Flags().StringArrayVar(&flags.ignore, ignoreFlagName, []string{}, "Files to ignore. Can be repeated")
 }
 
 func deprecationsCmdRun(cmd *cobra.Command, args []string) error {
-	schema, err := sources.LoadSchema(flags.schemaFile)
-	if err != nil {
-		return err
-	}
+	out := output.Data{}
 
-	queryFiles, err := input.ExpandGlobs(args, flags.ignore)
-	if err != nil {
-		return fmt.Errorf("Error: %s", err)
-	}
+	for _, schemaFile := range flags.schemaFiles {
+		schema, err := sources.LoadSchema(schemaFile)
+		if err != nil {
+			return err
+		}
 
-	queryFields, err := sources.LoadQueries(schema, queryFiles)
-	if err != nil {
-		return fmt.Errorf("Unable to parse files: %s", err)
+		queryFiles, err := input.ExpandGlobs(args, []string{})
+		if err != nil {
+			return fmt.Errorf("Error: %s", err)
+		}
+
+		queryFields, err := sources.LoadQueries(schema, queryFiles)
+		if err != nil {
+			return fmt.Errorf("Unable to parse files: %s", err)
+		}
+
+		for _, q := range queryFields {
+			f := output.Field{
```

Comment:
If `Field` is shared between packages, instead creating different representations, wouldn't be better if we create another packages with this objects?

Reply:
Hm... there's probably some refactoring opportunities here... And maybe generating the different outputs for deprecation, unused, etc could also be a responsibility of "output". Will have a think, but not something I think we should do as part of this PR. 🤔